### PR TITLE
Updated example for new alerting engine

### PR DESCRIPTION
**Pull request by pmusolino-rms**
Using the example of port utilization caused errors when trying to use it with the new alert system.  Putting variables in curly braces and removing php calls resolved it.  Updating docs to reflect this.

Also removed examples with hash tags in the body of the template as pasting this in will cause syntax errors by the alerts script.

DO NOT DELETE THIS TEXT

#### Please note

> Please read this information carefully. You can run `./scripts/pre-commit.php` to check your code before submitting.

- [ ] Have you followed our [code guidelines?](http://docs.librenms.org/Developing/Code-Guidelines/)

#### Testers

If you would like to test this pull request then please run: `./scripts/github-apply <pr_id>`, i.e `./scripts/github-apply 5926`

### CODE REVIEW

File: doc/Alerting/Templates.md [hunk 1]
```diff
@@ -281,7 +280,6 @@ Alert-ID: {{ $alert->id }} <br>
 Rule: @if ($alert->name) {{ $alert->name }} @else {{ $alert->rule }} @endif <br>
 @if ($alert->faults) Faults:
 @foreach ($alert->faults as $key => $value)
-#{{ $key }}: {{ $value['string'] }}<br>
```
Comment (murrant) [inline doc/Alerting/Templates.md]: Why did you remove this one?

File: doc/Alerting/Templates.md [hunk 2]
```diff
@@ -116,7 +116,6 @@ Unique-ID: {{ $alert->uid }}
 Rule: @if ($alert->name) {{ $alert->name }} @else {{ $alert->rule }} @endif
 @if ($alert->faults) Faults:
 @foreach ($alert->faults as $key => $value)
-  #{{ $key }}: {{ $value['string'] }}
```
Comment (murrant) [inline doc/Alerting/Templates.md]: Why did you remove this?

File: doc/Alerting/Templates.md (hunk 1 above)
Comment (pmusolino-rms) [reply doc/Alerting/Templates.md]: The lines starting with hash tags cause the parser to throw a syntax error.  If they are supposed to be comments and you would like to keep the line then I can amend it to use the {{-- comment --}} which I think is the laravel comment formatting.

File: doc/Alerting/Templates.md (hunk 2 above)
Comment (pmusolino-rms) [reply doc/Alerting/Templates.md]: The lines starting with hash tags cause the parser to throw a syntax error.  If they are supposed to be comments and you would like to keep the line then I can amend it to use the {{-- comment --}} which I think is the laravel comment formatting.

Comment (murrant) [reply doc/Alerting/Templates.md]: I think the hash is supposed to be a literal hash in the result.  No special meaning.
Really odd that it would cause a syntax error.
